src/test/java/uk/gov/pay/ledger/util/fixture/QueueRefundEventFixture.java

```diff
@@ -4,24 +4,22 @@
 import com.amazonaws.services.sqs.AmazonSQS;
 import com.google.common.collect.ImmutableMap;
 import com.google.gson.GsonBuilder;
-import org.apache.commons.lang3.RandomStringUtils;
 import uk.gov.pay.ledger.event.model.Event;
 import uk.gov.pay.ledger.event.model.ResourceType;
 
-import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 
 public class QueueRefundEventFixture implements QueueFixture<QueueRefundEventFixture, Event> {
     private String sqsMessageId;
     private ResourceType resourceType = ResourceType.REFUND;
     private Long amount = 50L;
     private String gatewayAccountId = "123456";
-    private String resourceExternalId = RandomStringUtils.randomAlphanumeric(20);
-    private String parentResourceExternalId = RandomStringUtils.randomAlphanumeric(20);
+    private String resourceExternalId = "resource_external_id";
+    private String parentResourceExternalId = "parent_resource_external_id";
```

Comment:
it's too long. It can be only 26 characters long (and is 27) therefore the build is failing.

Reply:
🥇